tests/config/configured_rule.php

```diff
@@ -3,9 +3,13 @@
 declare(strict_types=1);
 
 use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+use Rector\Core\Configuration\Option;
 
 return static function (ContainerConfigurator $containerConfigurator): void {
     $services = $containerConfigurator->services();
+    $parameters = $containerConfigurator->parameters();
 
     $services->set(\ViewScopeRector\ViewScopeRector::class);
+
+    $parameters->set(Option::AUTOLOAD_PATHS, [__DIR__.'/../../myautoload.php']);
```

Comment:
registration of my custom autoload file

Reply:
This works differently now. To include the file, use `BOOTSTRAP_FILES` instead

Reply:
But instead of these, the autoload path should include the controller class

Reply:
adjusted to use `BOOTSTRAP_FILES`, but this did not change the outcome of the repro.

Reply:
I tried settig `$parameters->set(Option::AUTOLOAD_PATHS, [__DIR__.'/../../builtin/controllers/']);` but it still leads to the same error